doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -127,73 +127,75 @@ MMIO Registers Definition
 Usage
 *****
 
-For usage information, see :ref:`enable_ivshmem`
+For usage information, see :ref:`enable_ivshmem`.
 
 Inter-VM Communication Security Hardening (BKMs)
 ************************************************
 
 As previously highlighted, ACRN 2.0 provides the capability to create shared
```

Comment:
@NanlinXie @dbkinder How can we update the paragraph, so that we don't need to mention a specific release, such as 2.0?

Reply:
Sure.  Historically, releases before 2.0 didn't support ivshmem but now we do:
```
ACRN lets you create shared ...
```